addons/l10n_dk/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.depends('vat', 'country_id')
+    def _compute_company_registry(self):
+        # OVERRIDE
+        # In Denmark, if you have a VAT number, it's also your company registry (CVR) number
+        super()._compute_company_registry()
+        for partner in self.filtered(lambda p: p.country_id.code == 'DK' and p.vat):
```

Comment:
People can forget to put DK in front of the vat number...

Reply:
> People can forget to put DK in front of the vat number...

No, they can't forget: There will be an explicit UserError thrown telling them what a DK vat number should look like.

> And why not suppose that if it is not set it is the same as the VAT?

That's exactly what we do with this piece of code. We just autocomplete it on save.

Reply:
@Levizar No, if you do not put DK at the start, it should fall back: https://github.com/odoo/odoo/blob/ea8a809868f1d1d2a26333a377adc0af446cff37/addons/base_vat/models/res_partner.py#L224
![Screenshot from 2023-07-05 18-25-35](https://github.com/odoo/odoo/assets/3832699/1d204506-0bb9-4e9e-b6c5-ab9798da13ca)